### PR TITLE
Keyword `examples`

### PR DESCRIPTION
**Pull request by AgniveshChaubey**
This PR addresses #76.

### CODE REVIEW

File: content/2020-12/meta-data/examples.markdown [hunk 1]
```diff
@@ -26,3 +26,134 @@ Annotations
 -----------
 
 This keyword produces the list of examples as the annotation value.
+
+## Explanation
+
+The `examples` keyword is used to provide a list of example instances associated with a particular schema that should ideally validate against the schema. These examples serve to illustrate the intended structure and constraints defined by the schema. While these examples are not used for validation purposes, they are helpful in providing sample valid instances against the schema they are defined in.
+
+_**Note:** While it is recommended that the examples should validate against the subschema they are defined in, this requirement is not strictly enforced._
```
Comment (jviotti) [inline content/2020-12/meta-data/examples.markdown:34]:
```suggestion
_**Note:** While it is recommended that the examples validate against the subschema they are defined in, this requirement is not strictly enforced._
```

File: content/2020-12/meta-data/examples.markdown [hunk 2]
```diff
@@ -26,3 +26,134 @@ Annotations
...
+  "then": {
+    "properties": {
+      "bar": {
+        "type": "array",
+        "examples": [ [ "foo" ], [ "bar", "baz" ] ]
+      }
+    }
+  },
+  "else": {
+      "properties": {
```
Comment (jviotti) [inline content/2020-12/meta-data/examples.markdown:80]: Indentation seems a bit off here

File: content/2020-12/meta-data/examples.markdown [hunk 3]
```diff
@@ -26,3 +26,134 @@ Annotations
...
+{ "foo": "foo", "bar": [ "bar" ] }
+{{</instance-pass>}}
+
+{{<instance-annotation>}}
+[
+  /// ...
+  {
+    "valid": true,
+    "keywordLocation": "/then/properties/bar/examples",
+    "instanceLocation": "",
```
Comment (jviotti) [inline content/2020-12/meta-data/examples.markdown:100]: Like on the `deprecated` PR, the instance location is incorrect, as it should be `/bar`. Can you go through the other `instanceLocation` to quickly check if they are all OK? Maybe we missed some other ones

Comment (AgniveshChaubey) [reply content/2020-12/meta-data/examples.markdown:100]: Sure!

**Review by jviotti — APPROVED** [review]

**Pull request merged**